src/Network/Network/ChangeLog.md

```diff
@@ -23,6 +23,9 @@
 * Added `DdosProtectionPlan` property in `AzPublicIpAddress`
 * Updated mapping in `AzPublicIpAddress` to always show/create DdosSettings
 * Fixed a bug that added Ddos related properties when viewing PublicIpAddress and DdosProtectionPlan objects
+* Fixed a Bug for Set-AzIpGroup cmdlet to support the -whatif parameter
```

Comment:
Redundant lines got added after resolving merge-conflicts. Please check

Reply:
removed the redundant lines